ProcessLib/InitialCondition.h

```diff
@@ -70,7 +70,8 @@ class MeshPropertyInitialCondition : public InitialCondition
 		assert(_property.getMeshItemType() == MeshLib::MeshItemType::Node);
 	}
 
-	virtual double getValue(MeshLib::Node const& n) const override
+	virtual double getValue(MeshLib::Node const& n,
+	                        int const component_id) const override
```

Comment:
OK, I thought you dropped the need for reading tuples from your concept, but we can stick to that need.